public/data/ThePyroTF2.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "Devin Droddy",
+    "type": "personal",
+    "bio": "Student engineer and aspiring programmer",
+    "avatar": "https://github.com/ThePyroTF2.png",
+    "links": [
+      {
+        "name": "GitHub",
+        "url": "https://github.com/ThePyroTF2",
+        "icon": "github"
+      },
+      {
+        "name": "Twitter",
+        "url": "https://twitter.com/JustDeeevin",
+        "icon": "twitter"
+      },
+      {
+        "name": "Youtube",
```

Comment:
I recommend capitalizing the `t` here. What do you think? 🙂
```suggestion
        "name": "YouTube",
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
I don't usually capitalize the `t` in YouTube but I do realize that's its official spelling. I'll make the change.

Good suggestion, thanks :)